packages/contentful/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@asap-hub/contentful",
+  "private": true,
+  "version": "0.1.0",
+  "engines": {
+    "node": "16.x"
+  },
+  "module": "build/index.js",
+  "main": "build-cjs/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {
+    "build:babel": "../../scripts/build-babel.sh build",
+    "watch:babel": "../../scripts/build-babel.sh watch",
+    "schema:export": "contentful space export --skip-content=true --space-id=$CONTENTFUL_SPACE_ID --environment-id=$CONTENTFUL_ENVIRONMENT --content-file=./schema/schema.json",
+    "schema:import": "contentful space import --content-model-only=true --space-id=$CONTENTFUL_SPACE_ID --environment-id=$CONTENTFUL_ENVIRONMENT --content-file=./schema/schema.json"
```

Comment:
Should we start prefixing/suffexing these things with crn/gp2? Even the schema location in the package.